### ISSUE TITLE
[css-properties-and-values-api] Resolved value of custom properties.

### ISSUE THREAD

**Issue by emilio** [w3c/csswg-drafts#10371]
The WPTs for this disagree on what the resolved value (getComputedStyle) of custom properties is when the computed value is currentcolor / color-mix() of currentcolor:

registered-property-computation.html says that it should return the value including `currentcolor`:

  https://github.com/web-platform-tests/wpt/blob/efe3de7191/css/css-properties-values-api/registered-property-computation.html#L161

at-property-animation says that it should be resolved:

  https://github.com/web-platform-tests/wpt/blob/efe3de7191/css/css-properties-values-api/at-property-animation.html#L440-L452

The computed value above is `color-mix(in srgb, currentcolor, rgb(200, 200, 200))`. Which one is it? Am I missing something? What is Blink / WebKit doing?

cc @flackr

**Comment by emilio** [comment]
I agree the computed value should include currentColor, the question is whether getComputedStyle should resolve it (like it does for any non-custom color property)

**Comment by svgeesus** [comment]
removing css-color-5 tag as the changes are to properties and values api

**Comment by benface** [comment]
This seems like a web-breaking change. [Here's a Codepen](https://codepen.io/benface/pen/KwddNvX) that renders differently in Safari TP 223, [which implemented this change](https://github.com/WebKit/WebKit/commit/0fb43e3086742cde9ab118f669c8c9ba28ee0790). In all other modern browsers released in the past 2 years, the circle is blue, but in Safari TP 223, it is black. cc @weinig @nt1m

**Comment by emilio** [comment]
@benface that seems like a Safari bug, independent of this change. Resolved values are only exposed via `getComputedStyle`.

**Comment by svgeesus** [comment]
@tabatkins @astearns @gregwhitworth it seems the [resolution](https://github.com/w3c/csswg-drafts/issues/10371#issuecomment-2162491973) clear enough to apply the edits? @emelio maybe send a PR?

**Comment by weinig** [comment]
Yeah, this is definitely just a bug in WebKit. Filed https://bugs.webkit.org/show_bug.cgi?id=296134.
